tensorboard/webapp/plugins/plugins_module.ts

```diff
@@ -19,16 +19,10 @@ import {PluginsContainer} from './plugins_container';
 import {PluginsComponent} from './plugins_component';
 import {CoreModule} from '../core/core_module';
 import {PluginRegistryModule} from './plugin_registry_module';
-import {PluginApiHostModule} from '../../components/experimental/plugin_util/plugin_api_host_module';
```

Comment:
Please remove plugin_util:plugin_host dependency in the BUILD file of this module.

Reply:
The dependency still exists in plugins_component.ts.